docs/release_notes/release_notes.md

```diff
@@ -14,7 +14,27 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 
 ## Features
 
+### Scenario Installer
+
+### OSTree Content
+
+### Lazy Sync
+
+### Docker V2
+
+### Mirror On Sync
+
+### Remote Execution Integration
+
+### Capsule Status on Proxy
+
+### Inter-Server Sync
```

Comment:
Users may now perform exports and imports of yum repositories to disk, either as a series of directories or as ISO files. This is especially useful for users in "air-gapped" network envionrments, where a Katello server might not have access to the Internet. Note that Inter-Server Sync (ISS) replaces the `katello-disconnected` tool. Additional information is available at http://www.katello.org/docs/user_guide/disconnected/.